readthedocs/projects/static-src/projects/js/import.js

```diff
@@ -237,6 +247,17 @@ function ProjectImportView (instance, config) {
             id = null;
         }
         self.filter_org(id);
+        self.filter_own(false);
+        self.page_current(null);
+    };
+
+    self.set_filter_own = function () {
+        // TODO: Since we are modifying three observable items this
+        // trigger the request three times (because of the computed
+        // value). How I can fix it?
```

Comment:
I these two functions I have the same problem. I'm modifying these observable objects that are used by the computed value, so I'd like to know if there is a way to modify all of them in an atomic way and launch just one computation.